www/app/Chat.tsx

```diff
@@ -55,6 +55,9 @@ async function fetchStream(
     });
 
     if (!response.ok) {
+      if (response.status === 402) {
+        return response;
```

Comment:
Shouldn't return anything in the case of an error just make a sweet alert directly